lib/pg_search/scope_options.rb

```diff
@@ -90,10 +90,17 @@ def subquery
     end
 
     def conditions
-      config.features
-        .reject { |_feature_name, feature_options| feature_options && feature_options[:sort_only] }
-        .map { |feature_name, _feature_options| feature_for(feature_name).conditions }
-        .inject { |accumulator, expression| Arel::Nodes::Or.new(accumulator, expression) }
+      expressions =
+        config.features
+          .reject { |_feature_name, feature_options| feature_options && feature_options[:sort_only] }
+          .map { |feature_name, _feature_options| feature_for(feature_name).conditions }
+
+      # https://github.com/rails/rails/pull/51492
+      if ActiveRecord.version >= Gem::Version.new("7.2.0.beta1")
```

Comment:
This is great! I'm wondering, should we change this to check Arel's version instead?

Reply:
No, because the arel version hasn't changed in 6 years (from git blame).